MechJeb2/MechJebModuleAirplaneAutopilot.cs

```diff
@@ -138,26 +143,27 @@ public void DisableAltitudeHold ()
         {
             if (!enabled)
                 return;
+
             AltitudeHoldEnabled = false;
             DisableVertSpeedHold ();
         }
 
-        private double convertAltitudeToVerticalSpeed (double deltaAltitude)
+        private double convertAltitudeToVerticalSpeed (double deltaAltitude, double reference = 5)
         {
-            double reference = 5;
             if (reference < 2)
                 reference = 2;
+
             if (deltaAltitude > reference || deltaAltitude < -reference)
                 return deltaAltitude / reference;
             else if (deltaAltitude > 0.1)
-                return deltaAltitude * deltaAltitude / (reference * reference);
-            else if (deltaAltitude > -0.1)
-                return 0;
+                return Math.Max(0.05, deltaAltitude * deltaAltitude / (reference * reference));
```

Comment:
could use Math.Sign() & Math.Abs() and merge both branches together, for clarity.